### DISCUSSION TITLE
Implement WotLK Arena Points System

### DISCUSSION THREAD

**Discussion by minthes** [chromiecraft/chromiecraft#6009] [Suggestions]
Currently, with an average team rating of 900, players earn around 160 points weekly. This setup can feel unrewarding and lacks a sense of progression. As it would take a whole season to get one or two upgrades.

Transitioning to the WotLK Arena Points system would provide a balanced solution. With an average of 630 points weekly, players experience a more gratifying progression curve. 

The transition to the WotLK system would provide players with a steady sense of progression. Earning around 630 points per week strikes a balance between making rewards attainable without diminishing their value. This setup aligns well with the evolving expectations of our community.

The proposed system allows players to earn upgrades at a consistent pace, ensuring a fair and balanced competitive scene. It also has diminishing returns so high rated teams won't get geared too fast and lose sense of progression.

Overall, this change will encourage sustained participation and reduce player frustration.

Lastly, I will leave you with a link to the blizzlike arena points calculator for wotlk: https://tbc.wow-tools.com/tbc_arena_calculator

**Comment by balleny** [comment]
i guess what is missing here is a proper write-down of the different systems, and what we have right now. otherwise, no one will even attempt to fix it.

### Original tbc / wotlk
1. teams starting at 1500, quite linear arena point rewards from 0 to 1500, then nonlinear
2. starting season 3, formulas were changed. they are listed in here and also visualized https://wowwiki-archive.fandom.com/wiki/Arena_point
3. starting s6, arena teams started at rating 0 instead of 1500, but teams below 1500 rating received points as if they were at 1500

### classic 
they started with a system, afaik starting also from 0 (but i couldnt confirm that yet) and adjusted the formulas in s4
https://www.reddit.com/r/classicwow/comments/us1cg5/tbc_arena_calculator_formula_updated_for_s4/


### Chromiecraft right now
starting at 0, but the arena points system similar original TBC (linear, not virtually ranking teams below 1500 rating as 1500 rating for point calculation)

**Comment by Reily93** [comment]
https://tbc.wow-tools.com/tbc_arena_calculator maybe this can be helpfull for the formulas of the woltk system

**Comment by shaszun** [comment]
With so little teams playing + bad matchmaking (getting matched against 1 of the top teams, around 1500 rating at that time 10 times in a row, when we were at 0) + bad balance in arenas (this is a long topic). I think this is a good suggestion.
